src/client/app/avdelingsleder/bemanning/components/sletteSaksbehandlerModal.less

```diff
@@ -10,7 +10,7 @@
 
 .buttons {
   display: inline-flex;
-  marin-top: 20px;
+  margin-top: 20px;
```

Comment:
Pga .text (som är texten over knapperna) har margin-bottom: 30px så blir det for mye med 20px till. Må egentligen bare fjerne margin-top her. 

```suggestion
```

Reply:
Fikset i https://github.com/navikt/k9-los-web/pull/622/commits/573b28abb16d8530425ca22550226ed5ebb7c330